### DISCUSSION TITLE
QUESTION: How to restrict access to one or more IP's

### DISCUSSION THREAD

**Discussion by ecxod** [gogs/gogs#7076] [Q&A]
I want to restrict the access to gogs to some IP's.  
I could do it using a reverse proxy, with something like this:
```
<Directory ${document}/>
  Options +Indexes +FollowSymLinks -MultiViews
  .....
  Require ip 12.23.34.45
</Directory>
```
But I do not want to install a proxy just for this issue.
Is there a way to do it directly from gogs? Maybe from app.ini?

**Comment by unknwon** [comment]
Hey, please use reverse proxy for this type of purpose, I think that is better.
